docs/docs/userguide/INSTALLATION.md

```diff
@@ -2,76 +2,83 @@
-    You can use a custom configuration file, but it must follow the same format as the default configuration file. You can make a copy of `config.tfvars`, using it as a template to define your infrastructure configuration.
-    
-!!! Warning "Use the same configuration file for uninstallation and cleanup"  
+??? tip "Can I use a custom configuration file?"
+    You can use a custom configuration file, but it must follow the same format as the default configuration file. You can make a copy of `config.tfvars`, renaming the copy and using `config.tfvars` as a template to define your own infrastructure configuration.
+
+??? Warning "Use the same configuration file for uninstallation and cleanup"  
     If you have more than one environment, make sure to manage the configuration file for each environment separately. When cleaning up your environment, use the same configuration file that was used to create it originally.
 
 ## 4. Run the installation script
 
-The installation script provisions the environment infrastructure and installs the selected products based on the passed configuration file.
+Based on how `config.tfvars` has been configured the installation script will 
+
+1. Provision the environment and infrastructure 
+2. Install the selected DC product(s) 
 
-The installation is unattended and invokes Terraform to handle the creation and management of the Kubernetes infrastructure. To keep track of the current state of the resources and manage changes, Terraform creates an S3 bucket to store the current state of the environment. A DynamoDB table is created to handle the locking of remote state files during the installation, upgrade, and cleanup stages to prevent the environment from being modified by more than one process at a time. 
- 
-The installation script is located in the root folder of the project.
+Installation is fully automated and requires no user intervention. Terraform is invoked under the hood which handles the creation and management of the Kubernetes infrastructure.
+
+
+??? info "Terraform deployment details"
+    To keep track of the current state of the resources and manage changes, Terraform creates an [AWS S3 bucket](https://aws.amazon.com/s3/){.external} to store the current state of the environment. An [AWS DynamoDB](https://aws.amazon.com/dynamodb/) table is created to handle the locking of remote state files during the installation, upgrade, and cleanup stages to prevent the environment from being modified by more than one process at a time. 
+
+    The installation script, `install.sh`, is located in the root folder of the project.
 
 Usage:  
 
 ```shell
-./install.sh [-c <config-file>] [-h]
+./install.sh [-c <config_file_path>] [-h]
 ```
 
 The following options are available:
 
-- `-c <config_file>` - Pass a custom configuration file when provisioning multiple environments
+- `-c <config_file_path>` - Pass a custom configuration file when provisioning multiple environments
```

Comment:
Also we need to warn to not using `-t` in `uninstall` command if there are any other environment is installed in the same region with the same account. Because `-t` will remove the s3 and dynamodb permanently which those are responsible for maintaining the state for all environment in the same region and created by the same account. 
Otherwise the remaining environments could not be managed by terraform.